### DISCUSSION TITLE
Get Medias filtered by Hashtag

### DISCUSSION THREAD

**Discussion by ca-dsgn** [pgrimaud/instagram-user-feed#229] [Q&A] [answered]
First of all, I love your code. I used the Instagram API in the past and it became more and more a pain in the ass to use it. Then I found your code on github and it solves all of my problems :-)

But now I have a new problem: Is it possible to get medias by a user profile, which have a specific hashtag? My idea was to fetch more and more of the users media and then afterwards filter them by hashtag (which wouldn't be so nice for the performance). This is why I thought to ask you if there is a better solution for this with your code.

**Comment by pgrimaud** [comment, marked as answer]
> Is it possible to get medias by a user profile, which have a specific hashtag? 

Unfortunately no. I found there was a way using the Instagram official API. But this endpoint seems to be not used on the instagram website or app. I can't find a way to filter hashtag of a user on website or mobile app.
